srv/salt/ceph/monitoring/prometheus/files/prometheus.yml.j2

```diff
@@ -13,6 +13,11 @@ rule_files:
 
 scrape_configs:
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  {% for config in salt['pillar.get']('monitoring:prometheus:custom_scrape_configs', {}) %}
+  {%- for key, value in config.items() %}
+  {%- if loop.first %}- {% else %}  {% endif %}{{- key }}: {{ value | json}}
+  {% endfor %}
+  {%- endfor -%}
```

Comment:
What is the function of the `{%-` and `-%}` delimeters?

Reply:
It's a Jinja thing which strips whitespace from the output